packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,6 +48,38 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
+:::tip
+
+## Use `<ErrorBoundary/>` fallback props without prop drilling
+
+### useErrorBoundaryFallbackProps
```

Comment:
I think we need to seperate component's doc and hook's doc.

I think you intended combine this hook with component document, because it is related to this component.
But I think it is more convenient to seperated component document  with hook document

reference is below
<img width="401" alt="image" src="https://github.com/suspensive/react/assets/57122180/843de792-cece-4f1e-b1f2-83ba393960e4">

Chakra's intend is like that,
> If you need to use hook, check this [link](https://chakra-ui.com/docs/components/checkbox/usage#hooks)

<img width="1143" alt="image" src="https://github.com/suspensive/react/assets/57122180/b990e870-a0ff-435a-b6df-0031ce38efb6">

But our document is mixed.

Reply:
I made https://github.com/suspensive/react/issues/220 to resolve it. Let's talk about it after this Pull Request